go.mod

```diff
@@ -71,6 +70,8 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
 
+require github.com/ory/dockertest/v3 v3.8.1
```

Comment:
[nitpick] We could have this `require` where it belongs (e.g. up if direct dependency, down if indirect).

Reply:
This was added automatically during `go install`, but I can put it with the rest of them